modules/nf-neuro/image/powderaverage/main.nf

```diff
@@ -21,7 +21,7 @@ process IMAGE_POWDERAVERAGE {
     def prefix = task.ext.prefix ?: "${meta.id}"
 
     def b0_thr = task.ext.b0_thr ? "--b0_thr ${task.ext.b0_thr}" : ''
-    def shells = task.ext.shells ? "--shells ${task.ext.shells}" : ''
+    def shells = task.ext.shells ? "--shells ${task.ext.shells}" : "--shells \$(cut -d ' ' --output-delimiter=\$'\\n' -f 1- $bval | awk -F' ' '{v=int(\$1)}{print v}' | uniq)"
```

Comment:
The scilpy script is doing this job itself, when no shell argument is supplied. Why do we need this ?

Reply:
I can't remember exactly why (I should have noted it somewhere), but this fix worked. I'll try to reproduce the error and see what happens. Let's wait for merging in the meantime!

Reply:
Well, I've tried to reproduce the error again, but it seems to run fine now. I reverted the change @AlexVCaron !